consensus/src/quorum_store/quorum_store.rs

```diff
@@ -167,6 +169,7 @@ impl QuorumStore {
         &mut self,
         fragment_payload: Data,
         expiration: LogicalTime,
+        digest_rx: DigestReturnChannel,
```

Comment:
This is Sender<_> so should be called digest_tx?

Reply:
yes:)